roxie/ccd/ccdprotocol.cpp

```diff
@@ -283,13 +283,13 @@ class ProtocolSocketListener : public ProtocolListener
             Owned<ISecureSocket> ssock;
             if (client)
             {
-                if (protocol && streq(protocol, "ssl"))
+                if (protocol.get() && streq(protocol.get(), "ssl"))
                 {
```

Comment:
yes, and if you do you don't need to first check protocol.get().
`if (streq(protocol.str(), "ssl"))`
is all you need.